CHANGELOG.md

```diff
@@ -1,5 +1,54 @@
 # Change Log
 
+## [0.8.0](https://github.com/confluentinc/ksql/releases/tag/v0.8.0-ksqldb) (2020-03-18)
+
+### Features
+
+* add ARRAY_LENGTH UDF ([#4725](https://github.com/confluentinc/ksql/pull/4725)) ([31a9d9d](https://github.com/confluentinc/ksql/commit/31a9d9d))
+* add confluent-hub to ksqlDB docker image ([#4729](https://github.com/confluentinc/ksql/pull/4729)) ([b74867a](https://github.com/confluentinc/ksql/commit/b74867a))
+* basic auth for new API server ([#4685](https://github.com/confluentinc/ksql/pull/4685)) ([5dcea24](https://github.com/confluentinc/ksql/commit/5dcea24))
+* Integrate new API with existing security framework ([#4713](https://github.com/confluentinc/ksql/pull/4713)) ([0f7de31](https://github.com/confluentinc/ksql/commit/0f7de31))
+* add the topic name to deserialization errors ([#4573](https://github.com/confluentinc/ksql/pull/4573)) ([0f7edf6](https://github.com/confluentinc/ksql/commit/0f7edf6))
+* display errors in CLI in red text ([#4509](https://github.com/confluentinc/ksql/pull/4509)) ([56f9c9b](https://github.com/confluentinc/ksql/commit/56f9c9b))
+* enhance `PRINT TOPIC`'s format detection ([#4551](https://github.com/confluentinc/ksql/pull/4551)) ([8b19bc6](https://github.com/confluentinc/ksql/commit/8b19bc6))
+* Integrate pull queries in new API([#4498](https://github.com/confluentinc/ksql/pull/4498)) ([cd7c058](https://github.com/confluentinc/ksql/commit/cd7c058))
+* introduce JSON_SR format ([#4596](https://github.com/confluentinc/ksql/pull/4596)) ([daa04d2](https://github.com/confluentinc/ksql/commit/daa04d2))
+* log groupby errors to processing logger ([#4575](https://github.com/confluentinc/ksql/pull/4575)) ([b503d25](https://github.com/confluentinc/ksql/commit/b503d25))
+* Plug insert streams into backend ([#4512](https://github.com/confluentinc/ksql/pull/4512)) ([2482f0f](https://github.com/confluentinc/ksql/commit/2482f0f))
+* Provide upper limit on number of push queries ([#4581](https://github.com/confluentinc/ksql/pull/4581)) ([2cd66c7](https://github.com/confluentinc/ksql/commit/2cd66c7))
+* Simple perf tool for new API ([#4590](https://github.com/confluentinc/ksql/pull/4590)) ([ee7ca8a](https://github.com/confluentinc/ksql/commit/ee7ca8a))
+* support custom column widths in cli ([#4616](https://github.com/confluentinc/ksql/pull/4616)) ([42282cd](https://github.com/confluentinc/ksql/commit/42282cd))
+* support JSON schema inference in ksqlDB ([#4563](https://github.com/confluentinc/ksql/pull/4563)) ([fcd6c7f](https://github.com/confluentinc/ksql/commit/fcd6c7f))
+* support Protobuf in ksqlDB ([#4469](https://github.com/confluentinc/ksql/pull/4469)) ([a77cebe](https://github.com/confluentinc/ksql/commit/a77cebe))
```

Comment:
@vcrfxia Can we nix "support Protobuf in ksqlDB (#4469) (a77cebe)" under 0.7.0 while we're at it? It looks like that was erroneously added a few releases ago.

Reply:
Good catch -- made the update. I think it happened because the PR was merged right after the 0.7.0 release was cut, before the changelog was generated. I'll make a note to be generate changelogs from commit hashes rather than directly from `master` to avoid this in the future.